xapian-applications/omega/templates/query

```diff
@@ -78,8 +78,8 @@ $set{clickurl,$if{$opt{clicklog},$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&D
 $if{$and{$field{modtime},$ne{$field{modtime},-1}},
 <script type="text/javascript">write_date($add{$field{modtime}});</script>
 <noscript>
-<span title="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
-Modified:<br><b>$html{$date{$field{modtime},%Y-%m-%d}}</b></span>
+<time title="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
```

Comment:
You don't have a `datetime` attribute here, although you do in the Javascript-generated one. Given you then want to replace the Javascript with something that reads that static datetime and replaces the element contents with a localised date string, this is important to get right. While you could read from `title`, that's intended for people to read (so we will probably end up with both — `title` and element contents that our Javascript will localised, and `datetime` which will have the full timezone, probably in UTC for simplicity).

Reply:
> You also don't need to merge master in. It just makes your branch more complex. (If you actually need changes more recent then you can rebase your changes onto latest master, which is generally how we merge PRs into master once ready.)

Though don't rebase and force push a branch already under review without discussion - it tends to make review more complicated.

Travis internally actually does a temporary merge of the PR branch with master and tests the result of that merge (which is helpful since that tests what the current result of merging the branch would actually be) so typically you only need to update to the current master if the branch stops merging cleanly.

Not all CI systems test a merged version though - e.g. gitlab's definitely just tests the branch, and I think appveyor is the same.